nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -122,35 +122,51 @@ class FunctionsToPatchWithoutTracing:
 class MagicFunctionsToPatch:
     MAGIC_FUNCTIONS_TO_PATCH = {
         NamespaceTarget.TORCH_TENSOR: [
+            "__abs__",
```

Comment:
Metatypes should be defined for all new operations and added support in NNCF algorithms.